<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlterarAluno</class>
 <widget class="QDialog" name="AlterarAluno">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>..::     Alterar Aluno     ::..</string>
  </property>
  <widget class="QLabel" name="lblId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtId">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblNome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>54</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNome">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>201</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblEndereco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Endere√ßo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtEndereco">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>331</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblTelefone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>61</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefone:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTelefone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLimpar">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LIMPAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAlterar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ALTERAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSair">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>SAIR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBuscar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemover">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVER</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
